infotv/frontend/postcss.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    plugins: {
+        autoprefixer: {
+            browsers: [
+                '>0.25%',
```

Comment:
This includes stuff like Safari 8 and Android Browser 4.4. For InfoTV something like `>= 1% in FI, Firefox ESR, not dead, not ie 11` should be enough.

Reply:
Apparently (#9) Tracon needs support for Android 4. I think Desucon runs this only in recent Chrome nowdays.